roles/edpm_ovn/defaults/main.yml

```diff
@@ -93,8 +93,7 @@ edpm_ovn_ovs_external_ids:
   rundir: "/var/run/openvswitch"
 
 # Set openvswitch other_config.
-edpm_ovn_ovs_other_config:
-  vlan-limit: 0
+edpm_ovn_ovs_other_config: {}
```

Comment:
to handle empty dict need to add when condition at https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_ovn/tasks/configure.yml#L69C5-L69C131 
can also add that to external_ids as same applies there too.